CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 1.6.0 (Unreleased)
+### Breaking Change
+In accordance with our [versioning policy](https://github.com/corretto/amazon-corretto-crypto-provider/blob/master/VERSIONING.rst),
+this release contains a low-risk breaking change. For details please see the [1.5.0](#150) section of this document.
+This change only impacts libraries that generate EC keys using the
```

Comment:
```suggestion
This change applies only to libraries that generate EC keys using the
```

Reply:
Thank you. As from side discussion this is an optional change so I think I'll keep it.